ansible/roles/worker/tasks/setup_docker_driver.yml

```diff
@@ -36,3 +36,6 @@
   file:
     name=/images
     state=directory
+
+- name: Make a base directory for state files of the provisioned instances
```

Comment:
That needs to be solved in a separate issue, for now it is supposed to be using the local storage. Hence, to make the system work we need that directory